covid_act_now/delphi_covid_act_now/run.py

```diff
@@ -97,3 +104,6 @@ def run_module(params):
         # Report failures: someone should probably look at them
         for exported_file in fails:
             print(f"Failed to archive '{exported_file}'")
+
+    elapsed_time_in_seconds = round(time.time() - start_time, 2)
+    logger.info("Completed indicator run", elapsed_time_in_seconds=elapsed_time_in_seconds)
```

Comment:
does it make sense to add any of the other summative stats? a la jhu:
https://github.com/cmu-delphi/covidcast-indicators/blob/0398be979c39e29cec2b719c96f0cbc91e7a7790/jhu/delphi_jhu/run.py#L171-L175

Reply:
good idea, i can add min and max exported dates. Though there is a weird pattern where it exports twice (one for positivity rate and one for total tests), so perhaps we need to split them up. Do we have any guidelines on when we want to use multiple logging statements vs bundling them all into one?

Reply:
The dates are the same for both exports, so you should be able to count them using something like
```
csv_export_count += 2*exported_csv_dates.size
```

Reply:
added